python/cudf/cudf/fuzz_tests/fuzz_test_parquet.py

```diff
@@ -38,5 +67,44 @@ def parquet_writer_test(gdf):
     assert_eq(actual, expected)
 
 
+@pythonfuzz(
+    data_handle=ParquetWriter,
+    params={
+        "row_group_size": np.random.random_integers(0, 10000, 100),
+        "compression": ["snappy", None],
+        "index": [True, False, None],
```

Comment:
Does parameterizing whether or not to write the index add any value? It's just another column to write and slightly changes the metadata, but I imagine we can provide sufficient test coverage for index handling in standard unit tests already.

Reply:
Removed the `index` param here for now. 

Also, FWIW we are not generating a random dataframe with a random index as well. I'll add in that change in a separate PR as we want to unblock other two PRs waiting for these changes.